app/services/discovery_report.rb

```diff
@@ -63,7 +63,7 @@ def process_dobj(dobj)
   # @return [Hash<Symbol => Boolean>] errors
   def registration_check(druid)
     begin
-      obj = Dor::Item.find(druid)
+      obj = Dor::Item.find(druid.druid)
```

Comment:
This doesn't match the documented signature of the method.  I.E., either need to:
- update line 62 about what is a valid parameter
- have line 58 call `registration_check(dobj.druid.druid)` (probably simplest)